src/Raven.Server/Documents/Indexes/MapReduce/AutoMapReduceIndex.cs

```diff
@@ -230,88 +197,77 @@ private Table GetMapEntriesTable(Transaction tx)
             return table;
         }
 
-        public unsafe void PutMappedResult(BlittableJsonReaderObject mappedResult, ReduceKeyState state, Table table, LazyStringValue documentKey, ulong reduceKeyHash)
+        private unsafe void PutMappedResult(BlittableJsonReaderObject mappedResult, LazyStringValue documentKey, ulong reduceKeyHash, ReduceKeyState state, Table mapEntriesTable, TransactionOperationContext indexContext)
         {
-            var etag = ++LastMapResultEtag; // TODO arek - it seems that etag it useless
-
-            var etagBigEndian = IPAddress.HostToNetworkOrder(etag);
-
-            var hashBigEndian = Bits.SwapBytes(reduceKeyHash);
-
             var tvb = new TableValueBuilder
             {
-                { (byte*) &etagBigEndian , sizeof (long) },
                 { documentKey.Buffer, documentKey.Size },
-                { (byte*) &hashBigEndian, sizeof(ulong) }
+                { (byte*) &reduceKeyHash, sizeof(ulong) }
```

Comment:
@ayende I removed the endianess conversion for `reduceKeyHash` value. Since there is no related index, I don't think that would be necessary. Is it correct?